tests/meta/upload/scout/test_generate_load_config.py

```diff
@@ -138,3 +142,27 @@ def test_generate_config_adds_case_paths(
 
     # THEN the config should contain the multiqc file path
     assert result_data.multiqc
+
+
+def test_generate_tomte_load_config(
+    tomte_analysis: Analysis,
+    upload_tomte_analysis_scout_api: UploadScoutAPI,
+    mocker: MockFixture,
+):
+    """Test that a Tomte config is generated."""
+
+    # GIVEN an analysis object that has been run with Tomte
+    assert tomte_analysis.workflow == Workflow.TOMTE
+
+    # GIVEN an upload Scout API with some Tomte information
+
+    # GIVEN a genome build
+    mocker.patch.object(TomteAnalysisAPI, "get_genome_build", return_value=GenomeVersion.HG19)
```

Comment:
Maybe it would be a good idea to mock this method directly in the `tomte_context` 🤔

Reply:
it would imply quite a few extra changes to clean up context, analysis and other fixtures. This needs to be done, but it's probably better to do in a separate PR

Reply:
Sorry! I misunderstood you then. I can move that line to the context. The context fixture needs clean up though, but that part we'll keep in a separate pr